docs/button.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button Component</title>
+    <link rel="stylesheet" href="/styles/main-style.css">
+    <link rel="stylesheet"
+    href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
+    <script src="https://kit.fontawesome.com/44c4f0d450.js" crossorigin="anonymous"></script>  
+</head>
+<body>
+    <nav class="navigation">
+        <a href="https://github.com/therajatg/parts"><h2 class="logo">parts</h2></a>
+        <div class="nav-links">
+            <span><a href="https://github.com/therajatg/parts">Docs</a></span><span><a href="https://github.com/therajatg/parts">Github</a></span>
+        </div>
+    </nav>
+
+    <div class="sidebar-and-description">
+        <div class="sidebar">
+            <h2>Components</h2>
+            <ul>
+                <li><a href="avatar.html">Avatar</a></li>
+                <li><a href="alert.html">Alert</a></li>
+                <li><a href="badge.html">Badge</a></li>
+                <li><a href="button.html">Button</a></li>
+                <li><a href="card.html">Card</a></li>
+                <li><a href="image.html">Image</a></li>
+                <li><a href="input.html">Input</a></li>
+                <li><a href="text-utilities.html">Text Utilities</a></li>
+                <li><a href="lists.html">Lists</a></li>
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="rating.html">Rating</a></li>
+                <li><a href="toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="simplified-grid.html">Simplified Grid</a></li>
+            </ul>
+        </div>
+        <div class="description">
+            <h3>Button</h3>
+            <h4>Buttons allow users to take actions, and make choices, with a single tap. Buttons are a quite frequent occurance in most websites  such as in e-commerce sites for buying an item or adding an item to the cart, in social media sites to follow or connect with someone or as call-to-action element in most fo the sites. Various types of buttons are as follows: </h4>
```

Comment:
Do you really want thr text to be bold? P tag would work here instead of h4

Reply:
Changed to </p>

Reply:
changed h4 to p